<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMsoftWorkbench</class>
 <widget class="QMainWindow" name="EMsoftWorkbench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMsoftWorkbench</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="styleSheet">
       <string notr="true">QTabWidget::tab-bar
{
	top: 5px;
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <widget class="QWidget" name="monteCarloTab">
       <attribute name="title">
        <string>Monte Carlo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6" rowstretch="0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="MPMCDisplayWidget" name="monteCarloDisplayWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="masterPatternTab">
       <attribute name="title">
        <string>Master Pattern</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="MPMCDisplayWidget" name="masterPatternDisplayWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="patternDisplayTab">
       <attribute name="title">
        <string>Pattern Display</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>-1</number>
        </property>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Angle Mode</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="generateBtn">
               <property name="text">
                <string>Generate</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="angleTypeCB">
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
             <item>
              <property name="text">
               <string>Single Angle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Read Angle File</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sample Cubochoric Space</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Use Sampling Rate</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QVBoxLayout" name="angleWidgetLayout"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
	font-weight: bold;
}</string>
          </property>
          <property name="title">
           <string>Detector and Microscope Geometry</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="5" column="1">
            <widget class="QLineEdit" name="pc1">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="scintillatorDist">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLineEdit" name="samplingStepSize2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="3">
            <widget class="QLineEdit" name="beamCurrent">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QLineEdit" name="scintillatorPixSize">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="detectorTiltAngle">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QLineEdit" name="detectorOmegaAngle">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="numOfPixelsY">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Scintillator Pixel Size (microns)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>by</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Detector Omega Angle (degrees)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Number of Camera Pixels (WxH)</string>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>,</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Pattern Center (pixels, xy)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Pixel Coordinates</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Energy Min - Max</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>,</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>,</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Dwell Time (mu s)</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Barrel Distortion (x10^-10)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Sampling Step Size</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Beam Current (nA)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Scintillator Distance (microns)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Detector Tilt Angle (degrees)</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QComboBox" name="energyMinCB"/>
           </item>
           <item row="11" column="3">
            <widget class="QComboBox" name="energyMaxCB"/>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="pixelCoords2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="numOfPixelsX">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="pc2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="pixelCoords1">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="samplingStepSize1">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="3">
            <widget class="QLineEdit" name="dwellTime">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="3">
            <widget class="QLineEdit" name="barrelDistortion">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="logTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveLogBtn">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShowUserManual">
   <property name="text">
    <string>Show User Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPMCDisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">MPMCDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scintillatorDist</tabstop>
  <tabstop>detectorTiltAngle</tabstop>
  <tabstop>detectorOmegaAngle</tabstop>
  <tabstop>scintillatorPixSize</tabstop>
  <tabstop>numOfPixelsX</tabstop>
  <tabstop>numOfPixelsY</tabstop>
  <tabstop>pc1</tabstop>
  <tabstop>pc2</tabstop>
  <tabstop>pixelCoords1</tabstop>
  <tabstop>pixelCoords2</tabstop>
  <tabstop>samplingStepSize1</tabstop>
  <tabstop>samplingStepSize2</tabstop>
  <tabstop>beamCurrent</tabstop>
  <tabstop>dwellTime</tabstop>
  <tabstop>barrelDistortion</tabstop>
  <tabstop>energyMinCB</tabstop>
  <tabstop>energyMaxCB</tabstop>
  <tabstop>angleTypeCB</tabstop>
  <tabstop>generateBtn</tabstop>
  <tabstop>logTextEdit</tabstop>
  <tabstop>saveLogBtn</tabstop>
  <tabstop>tabWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
